src/tespy/components/heat_exchangers.py

```diff
@@ -867,6 +867,26 @@ def calc_parameters(self):
 
         self.check_parameter_bounds()
 
+    def exergy_balance(self, Tamb):
+        r"""
+        Perform exergy balance of a simple heat exchanger.
+
+            .. math::
+
+                T_m = \frac{T_{ein} - T_{aus}}{\ln\left({\frac{T_{ein}}
+                                                         {T_{aus}}}\right)}
+                Ex_{output} = \dot{m}_{in} \cdot (ex_{ph,out} - ex_{ph,in})
+                Ex_{input} = \left(1 - \frac{T_{amb}}
+                                   {T_m}\right) \cdot \dot{Q}
+                Ex_{loss} = 0
+        """
+        Tm = (self.inl[0].T.val_SI - self.outl[0].T.val_SI) / (
+            np.log(self.inl[0].T.val_SI / self.outl[0].T.val_SI))
+        self.Ex_output = self.inl[0].m.val_SI * (self.outl[0].ex_physical
+                                                 - self.inl[0].ex_physical)
```

Comment:
Having the linebreak after the first parenthesis would be more consistent with the other line, I think.